src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -309,6 +330,32 @@ void MainPage::EnsureCalculator()
     }
 }
 
+void MainPage::EnsureAlwaysOnTopCalculator()
+{
```

Comment:
Why are you creating a separate calculator just for always-on-top mode? This creates a lot of additional (and duplicate) elements in the UI tree, which has a high potential for slowing down the app startup time and increasing the memory usage.

Can we examine if there's a way we can implement always-on-top using the existing UI tree and managing visual states, instead?